samples/interleave/sample.go

```diff
@@ -69,8 +69,9 @@ type Track struct {
 
 type Venue struct {
 	gorm.Model
-	Name        string
-	Description string
+	Name         string
+	Description  string
+	VenueDetails spanner.NullJSON
```

Comment:
nit: Can we also update our README file to indicate that this is the correct type to use for JSON columns: https://github.com/googleapis/go-gorm-spanner?tab=readme-ov-file#data-types